int32 camera_number
---
bool success
